cfme/services/catalogs/myservice.py

```diff
@@ -160,3 +163,9 @@ def check_vm_add(self, add_vm_name):
     def reconfigure_service(self):
         sel.force_navigate('reconfigure_service',
                            context={'service_name': self.service_name})
+
+    def test_download_file(self):
+        sel.force_navigate('all_service')
+        download_btn('Download as Text')
+        download_btn('Download as CSV')
+        download_btn('Download as PDF')
```

Comment:
Wouldn't it be better to have the Text/CSV/PDF parametrized for single test? I don't see it as very good to have a method named `test_` inside the model. Also it should be worth noting this is gonna work only for firefox, I remember Chrome cannot be set to download the file unattended so far. It might need something like `cfme/tests/intelligence/test_download_report.py:30` because I am not sure whether the file download dialog blocks in Chrome or not.